app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,61 @@
+class AlbumsController < ApplicationController
+  def index
+    @model = Album.all
+    @path = new_album_path
+    @add_button = "Add New Album"
+  end
+
+  def show
+    @media = Album.find(params[:id])
+    @path = upvote_album_path(@media)
+    @edit_path = edit_album_path(@media)
+    @index_button = "View all Albums"
+    @index_path = albums_path
+  end
+
+  def new
+    @model = Album.new
+    @action = "create"
+    @method = :post
+    @title = "New Album"
+    @artist_type = "Artist"
+  end
+
+  def create
+    Album.create(album_params)
+    redirect_to albums_path
+  end
+
+  def edit
+    id = params[:id]
+    @artist_type = "Artist"
+    @title = "New Album"
+    @action = "update"
+    @method = :patch
+    @model = Album.find(id)
```

Comment:
Since you're not using the variable `id` from line 30 again anywhere else, you could just do

```ruby
@model = Album.find(params[:id])
```